### ISSUE TITLE
issues on the examples given

### ISSUE THREAD

**Issue by guidofranceschini** [rbouqueau/TTML_in_MP4_DASH_statement#10]
Hi,

I read your examples, but I found one typo, and one more relevant issue.

The typo is in example 1 strategy 2: the sample 6-12 should have begin=10, not 6

The other issue is about absolute/relative timing in the begin/end tags: in your examples, timing refers to the start of the presentation. Is this really correct? I am using TTML segmented in MP4 files for Smooth Streaming, and times in begin/end tags had to be relative to the start of the sample (BTW: doing so, it works fine on all tested devices). I thought that TTML in MP4 for DASH would have been the same as TTML in MP4 for SS...

Guido

**Comment by andreastai** [comment]
Hi Guido,

Thanks for your comments.

> The typo is in example 1 strategy 2: the sample 6-12 should have begin=10, not 6

The samples in the example have a fixed duration of 6 seconds, so it is correct. 

> The other issue is about absolute/relative timing in the begin/end tags: in your examples, timing refers to the start of the presentation. Is this really correct? 

Yes, because the reference is ISO 14496-30 [1]. The timing in TTML documents are relative to the track presentation timeline, not the start of the sample.

[1] http://www.iso.org/iso/catalogue_detail.htm?csnumber=63107

**Comment by guidofranceschini** [comment]
Sorry, I wasn't clear about the example issue:
Sample 2 (6-12s)
[...]
            <p xml:id="subtitle1" region="bottom" begin="00:00:06.000" end="00:00:12.000" style="paragraphStyle">

should be instead:
Sample 2 (6-12s)
[...]
            <p xml:id="subtitle1" region="bottom" begin="00:00:10.000" end="00:00:12.000" style="paragraphStyle">

BR
Guido

**Comment by guidofranceschini** [comment]
Let me try again (xml didn't pass):

Sample 2 (6-12s)
[...]
begin="00:00:06.000" end="00:00:12.000"

should be instead:

Sample 2 (6-12s)
[...]
begin="00:00:10.000" end="00:00:12.000"

BR
Guido

**Comment by nigelmegitt** [comment]
Thanks Guido, I understand the issue you are raising: the first active span always only begins at 10s so there is no way that it can begin earlier regardless of the strategy chosen.

**Comment by nigelmegitt** [comment]
I have fixed that issue in place - since the examples are on the wiki there's no associated commit or pull request. @guidofranceschini @tairt you might want to have a look and verify this.

**Comment by andreastai** [comment]
Thanks @nigelmegitt and @guidofranceschini. This was indeed an error. Thanks for fixing it. I close the isssue but feel free to re-open it in case you think there is something outstanding.

**Comment by cconcolato** [comment]
@guidofranceschini 

> I thought that TTML in MP4 for DASH would have been the same as TTML in MP4 for SS...

This is very worrying as Microsoft pushed for the way TTML in MP4 is currently specified ... Can you raise the question on the MP4 mailing list (or CMAF, or both).

**Comment by nigelmegitt** [comment]
I think they possibly had learned from the Smooth experience, and there were also other inputs than Microsoft in that debate. I'm very comfortable that they ended up with the right answer for TTML in MP4 regardless of how it is done in Smooth, and would not want to reopen a debate that does not need to be opened.
